client/components/blank-canvas/index.tsx

```diff
@@ -56,12 +56,18 @@ BlankCanvas.Header = ( {
 	</div>
 );
 
-BlankCanvas.Content = ( { children }: PropsWithChildren ) => (
+const BlankCanvasContent = ( { children }: PropsWithChildren ) => (
 	<div className="blank-canvas__content">{ children }</div>
 );
 
-BlankCanvas.Footer = ( { children }: PropsWithChildren ) => (
+const BlankCanvasFooter = ( { children }: PropsWithChildren ) => (
 	<div className="blank-canvas__footer">{ children }</div>
 );
 
+const BlankCanvas = Object.assign( BlankCanvasComponent, {
+	Header: Object.assign( BlankCanvasHeader, { displayName: 'BlankCanvas.Header' } ),
+	Content: Object.assign( BlankCanvasContent, { displayName: 'BlankCanvas.Content' } ),
+	Footer: Object.assign( BlankCanvasFooter, { displayName: 'BlankCanvas.Footer' } ),
+} );
```

Comment:
Fixes [ratcheted lint errors](https://teamcity.a8c.com/viewLog.html?buildId=14831321&tab=Inspection&buildTypeId=calypso_calypso_WebApp_Check_Code_Style_Branch) (missing display name).

To smoke test:

On your site's `home` (`http://calypso.localhost:3000/home/{siteurl}`), scroll down to the educational cards, swipe to "Blog like an expert from day one" and click "Start learning". The modal dialog is the BlankCanvas.

<img width="694" alt="Educational cards" src="https://github.com/user-attachments/assets/07eb0a8c-a7e2-4c27-b6f3-41bc668538e3" />